website/docs/patterns/data-visualization/partials/guidelines/accessibility.md

```diff
@@ -0,0 +1,3 @@
+## Accessiblity considerations
```

Comment:
This content is really great! It doesn't need to happen now, but visual examples of charts we view as accessible may help users understand this section more.

Reply:
Thanks!

@jorytindall would you be interested in helping with this sometime? (I don't think it needs to be part of this initial PR though. Hoping to merge this one soon.)

Reply:
I'll make a task for it, it should be easy to cull together some examples, but I'd advocate for shipping this initial PR in the meantime

Reply:
I went ahead and added these in https://github.com/hashicorp/design-system/pull/2255/commits/cb539971e723a805820713e7a442af9c981c8d23

There are some "captions" that we could migrate in the future, open to feedback on the language!

Reply:
@jorytindall Added some great examples of accessible charts